HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -113,6 +121,17 @@ private FdaValidationResult ValidateTIFFiles(List<string> tifFiles, string direc
 
         #endregion
 
+        /// <summary>
+        /// Creates a VM row from the HydraulicProfile. This is used when editing an existing element.
+        /// </summary>
+        private static WaterSurfaceElevationRowItemVM CreateRowFromHydraulicProfile(IHydraulicProfile profile, string elementName)
+        {
+            string name = Path.GetDirectoryName(profile.FileName); //name is what's shown to the UI. For Gridded data, we want the directory name, not the file name. 
+            string path = Path.Combine(Connection.Instance.HydraulicsDirectory, elementName, profile.FileName); //path is the full path to the file from the element root.
+            double prob = profile.Probability;
```

Comment:
We actually save hydraulic profiles, not these rows to database, which is how it should be. So I'm just creating these rows from the saved data objects here.